geom/type_envelope.go

```diff
@@ -281,6 +281,7 @@ func (e Envelope) Distance(o Envelope) (float64, bool) {
 }
 
 // TransformXY transforms this Envelope into another Envelope according to fn.
+// TODO: remove the error here as well.
```

Comment:
Woops, missed this one before raising a PR.

- [ ] Remove error from `Envelope`'s `TransformXY` signature.